packages/runtime/src/templating/view.ts

```diff
@@ -59,13 +59,13 @@ export class View implements IView {
     this.$bind = lockedBind;
   }
 
-  public release(flags: LifecycleFlags): any {
+  public release(flags: LifecycleFlags): boolean {
     this.isFree = true;
     if (this.$state & State.isAttached) {
       return this.cache.canReturnToCache(this);
     }
 
-    return this.$unmount(flags);
+    return this.$unmount(flags) as boolean;
```

Comment:
```suggestion
    return !!this.$unmount(flags);
```
Would that work? Then we're not lying about the returned type as much :)